tests/pkg/fixtures/fomation_requests.go

```diff
@@ -8,6 +8,19 @@ import (
 	gcli "github.com/machinebox/graphql"
 )
 
+func FixGetFormationsForParticipantRequest(participantID string) *gcli.Request {
```

Comment:
I think it will be a good idea to generate an example for the new query under `examples` dir so that we have it by hand when we need it.